cmd/destroy.go

```diff
@@ -181,22 +182,44 @@ func destroyLab(ctx context.Context, c *clab.CLab) (err error) {
 		labDir = filepath.Dir(containers[0].Labels["clab-node-lab-dir"])
 	}
 
+	if maxWorkers == 0 {
+		maxWorkers = uint(len(containers))
```

Comment:
what is the idea behind casting to `uint` the length of a slice?

Reply:
`maxWorkers` was of `uint` type, I simply re-used the same var. TBH, I don't know why it has that type originally, maybe you would know?

Reply:
ah, yes, it is because cobra flag is of uint type - https://github.com/srl-labs/containerlab/blob/70351f84b7d26cd6e1d3ed5bc89e2db3e46edae0/cmd/deploy.go#L186

Thats ok